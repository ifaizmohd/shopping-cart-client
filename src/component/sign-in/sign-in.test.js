import { shallow } from "enzyme";
import React from "react";
import SignIn from "./sign-in.component";

describe("Signin component testing", () => {
  it("Shallow snapshot testing of Signin component", () => {
    expect(shallow(<SignIn />)).toMatchSnapshot();
  });
});
