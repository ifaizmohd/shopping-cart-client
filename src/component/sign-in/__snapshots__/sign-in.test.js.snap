// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signin component testing Shallow snapshot testing of Signin component 1`] = `ShallowWrapper {}`;
